src/services/oss/uri.rs

```diff
@@ -41,7 +41,7 @@ pub(super) fn percent_encode_path_hard(path: &str) -> String {
 
 #[cfg(test)]
 mod tests {
-    use super::*;
+    use super::percent_encode_path_hard;
```

Comment:
We still need this? Can we use `http_util::percent_encode_path`?

Reply:
when listing, the forward slashes `/` in `delimiter` and `prefix` should be encoded. But `http_util::percent_encode_path` will ignore them.

Reply:
OK, let's refactor them later.